src/pages/login.jsx

```diff
@@ -2,9 +2,8 @@ import React, { useState } from "react";
 import styled from "styled-components";
```

Comment:
Very good use of styled components - I love it 👍 , i typically break down e.g 

`const StyledButton = styled.button `